src/gui/hist_dialog.c

```diff
@@ -217,12 +217,14 @@ int dt_gui_hist_dialog_new(dt_gui_hist_dialog_t *d, int imgid, gboolean iscopy)
     do
     {
       dt_history_item_t *item = (dt_history_item_t *)items->data;
-
-      gtk_list_store_append(GTK_LIST_STORE(liststore), &iter);
-      gtk_list_store_set(GTK_LIST_STORE(liststore), &iter, DT_HIST_ITEMS_COL_ENABLED,
+      // This is sort of a hack, any better solution?
+      if (strcmp(item->op,"gamma") != 0)
```

Comment:
I was mostly concerned with history widget in darkroom, not copy-paste dialog;)

Reply:
Just in case nobody misses it :-)

Reply:
Maybe checking this in dt_history_get_items() ?

As this routine is used for both style dialog and history dialog we will avoid also gamma in the style dialog?

Reply:
BTW, a check for mask_manager is already done in  dt_history_get_items() so it seems a good place to put a second hack at the same place until a clean solution is found.